cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -182,8 +185,8 @@ public TestCommand() {
     @CommandLine.Option(names = "--native", description = "enable running test suite against native image")
     private Boolean nativeImage;
 
-    private static final String testCmd = "bal test [--offline]\n" +
-            "                   [<ballerina-file> | <package-path>] [(--key=value)...]";
+    private static final String testCmd = "bal test [--OPTIONS]\n" +
+            "                   [<ballerina-file> | <package-path>] [-- (-Ckey=value)...]\n ";
```

Comment:
Discussed with @hevayo and we concluded agreeing to have the following pattern for the test command:
`bal test -Ckey1=val1 -Ckey2=val2`
Please change accordingly.